_posts/2014-01-01-controller-web-api.md

```diff
@@ -0,0 +1,77 @@
+---
+layout: page
+title: "Web API Applications"
+category: controller
+date: 2013-06-06 08:55:36
```

Comment:
date is incorrect

Reply:
seems this date isn't actually used anywhere. this metadata section is consumed by jekyll which is mostly used for blogs where dates are used to sort blog posts. We're not using it in that way, so I removed all dates from the pages.